server/services/src/fileServer.ts

```diff
@@ -0,0 +1,131 @@
+//
+// Copyright © 2021 Anticrm Platform Contributors.
+//
+// Licensed under the Eclipse Public License, Version 2.0 (the 'License');
+// you may not use this file except in compliance with the License. You may
+// obtain a copy of the License at https://www.eclipse.org/legal/epl-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an 'AS IS' BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+import { S3Storage } from '@anticrm/s3'
+import { assignWorkspace, decodeToken } from '@anticrm/server'
+import { Account, generateId, Ref, Space } from '@anticrm/core'
+import Koa, { Context } from 'koa'
+import bodyParser from 'koa-bodyparser'
+import Router from 'koa-router'
+
+const S3_URI = process.env.S3_URI ?? 'http://127.0.0.1:9000'
```

Comment:
Why not https?

Reply:
If I could pass certificates to minio....